packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,15 @@
 <template>
   <template if:true={shouldDisplay}>
     <li class={tabClass} title={label} role="presentation">
-      <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
+      <button
+        class="slds-tabs_button slds-button"
+        onclick={select}
+        aria-label={arialLabelText}
```

Comment:
to represent which tab is currently selected, I would add `aria-pressed="true"`/`aria-pressed="false"`. Beware that `aria-pressed="false"` is not equivalent to omitting `aria-pressed`. The existence of `aria-pressed` with any value indicates that this is a toggle button, not a regular button.

Alternatively, you could replace the buttons with radio buttons ([doc](https://www.w3.org/TR/wai-aria-1.2/#radio)) to indicate that only one can be selected and you can't press an already selected button. However, that would require more changes since radio buttons are expected to support arrow keys and space bar ([doc](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role#keyboard_interactions)).

Reply:
Very useful, thanks @btaillon

Reply:
`aria-pressed` added. And thanks for the info!!!@btaillon